README.md

```diff
@@ -3,12 +3,22 @@
 [![Build Status](https://travis-ci.org/marco-c/crashsimilarity.svg?branch=master)](https://travis-ci.org/marco-c/crashsimilarity)
 [![codecov](https://codecov.io/gh/marco-c/crashsimilarity/branch/master/graph/badge.svg)](https://codecov.io/gh/marco-c/crashsimilarity)
 
-## Tests
+### Description
+Crashsimilarity is a tool to cluster slightly different crash reports, reporting the same problem. It is built for mozilla and support crash reports produced by [breakpad](https://chromium.googlesource.com/breakpad/breakpad/).
```

Comment:
> (link to socorro home page)

Should it be link to https://crash-stats.mozilla.com/ or to https://github.com/mozilla/socorro ?